package/src/utils/utils.ts

```diff
@@ -512,3 +513,16 @@ export const emojiRegex =
 
 export const urlRegex =
   /(?:\s|^)((?:https?:\/\/)?(?:[a-z0-9-]+(?:\.[a-z0-9-]+)+)(?::[0-9]+)?(?:\/(?:[^\s]+)?)?)/g;
+
+export const getMessagesComparatorCallback = <
```

Comment:
Can you remove `callback` from the name? This function will not be used as a callback.

Also better to add some documentation here.

Reply:
How about simply a `compareMessages`?

Reply:
✅

Reply:
✅